src/components/_file.scss

```diff
@@ -5,25 +5,25 @@
   align-items: center;
   gap: 1rem;
   min-width: 0;
-}
 
-.file .text-info {
-  width: 100%;
-  overflow: hidden;
-}
+  .text-info {
+    width: 100%;
+    overflow: hidden;
+  }
 
-.file .file-icon {
-  width: 2.3rem;
-  object-fit: cover;
-}
+  &-icon {
+    width: 2.3rem;
+    object-fit: cover;
+  }
 
-.file .file-name {
-  display: block;
-  color: map-get(variables.$colors, "heading");
-  line-height: 1.7;
-}
+  &-name {
```

Comment:
this compiles to `.file-name`. In html this probably looks smth like
```html
<div class="file">
   <span class="file-name">...</span>
</ div>
```
But we don't duplicate component names in its' children